kolibri/plugins/audio_mp3_render/assets/src/views/index.vue

```diff
@@ -155,7 +155,9 @@
         this.$refs.audio.currentTime = sum;
       },
 
-      updateTime() {
+      updateTime(e) {
+        // guard against the function being called after component is unmounted
+        if (!this.$refs.audio) return;
```

Comment:
not a big deal because it's all simple changes, but looks like this commit found its way into the wrong PR. (also in https://github.com/learningequality/kolibri/pull/1313)